src/index.ts

```diff
@@ -0,0 +1,542 @@
+import { AccessToken, ResourceOwnerPassword } from 'simple-oauth2'
```

Comment:
Here is your source code with some types added (I have to finish, help is welcome :))